src/main/java/teamrtg/rtg/api/tools/deco/DecoAbyssalCraftTree.java

```diff
@@ -0,0 +1,97 @@
+package teamrtg.rtg.api.tools.deco;
+
+
+import net.minecraft.init.Blocks;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.gen.feature.WorldGenerator;
+import net.minecraftforge.event.terraingen.TerrainGen;
+import teamrtg.rtg.api.tools.feature.tree.abyssalcraft.WorldGenTreeACDarkwood;
+import teamrtg.rtg.api.world.RTGWorld;
+import teamrtg.rtg.api.world.gen.RealisticBiomeGenerator;
+
+import java.util.Random;
+
+import static net.minecraftforge.event.terraingen.DecorateBiomeEvent.Decorate.EventType.TREE;
+
+/**
+ * 
+ * @author WhichOnesPink
+ *
+ */
+public class DecoAbyssalCraftTree extends DecoTree
```

Comment:
only the vanilla decos should go in `api.tools`. These should go in `module.abyssalcraft`